packages/contracts-bedrock/src/governance/ProposalValidator.sol

```diff
@@ -180,15 +199,21 @@ contract ProposalValidator is OwnableUpgradeable {
     /// @notice Initializes the ProposalValidator contract.
     /// @param _owner The address that will own the contract.
     /// @param _minimumVotingPower The minimum voting power required for a delegate to approve proposals.
-    /// @param _votingCycleBlock The block number of the current voting cycle.
+    /// @param _cycleNumber The number of the current voting cycle.
+    /// @param _startBlock The block number of the starting block of the voting cycle.
+    /// @param _duration The duration of the voting cycle.
+    /// @param _distributionLimit The max amount of tokens that can be distributed during the voting cycle.
     /// @param _distributionThreshold The max amount of tokens that can be distributed in a proposal.
     /// @param _proposalTypes Array of proposal types to set approval thresholds for.
     /// @param _requiredApprovals Array of approval thresholds corresponding to the proposal types.
     /// @param _immutableProposalTypeDatas Array of immutable proposal type data corresponding to the proposal types.
     function initialize(
         address _owner,
         uint256 _minimumVotingPower,
-        uint256 _votingCycleBlock,
+        uint256 _cycleNumber,
+        uint256 _startBlock,
+        uint256 _duration,
+        uint256 _distributionLimit,
```

Comment:
I would prefer a more descriptive name for this variable like we do with `votingCycleDistributionLimit`